lxd/vm_qemu.go

```diff
@@ -2413,14 +2413,21 @@ func (vm *vmQemu) RenderState() (*api.InstanceState, error) {
 					m["host_name"] = vm.localConfig[fmt.Sprintf("volatile.%s.host_name", k)]
 				}
 
+				hostCounters := shared.NetworkGetCounters(m["host_name"])
+
 				networks[k] = api.InstanceStateNetwork{
 					Addresses: addresses,
-					Counters:  api.InstanceStateNetworkCounters(shared.NetworkGetCounters(m["host_name"])),
-					Hwaddr:    m["hwaddr"],
-					HostName:  m["host_name"],
-					Mtu:       iface.MTU,
-					State:     "up",
-					Type:      "broadcast",
+					Counters: api.InstanceStateNetworkCounters{
```

Comment:
@stgraber perhaps a comment here explaining the counter-intuitive way the sent/recv fields are swapped (for our (my) future benefit)? :)

Reply:
done